csharp-advanced-topics/ActionAndFuncInCsharp/ActionAndFunc/ActionAndFunc.csproj

```diff
@@ -0,0 +1,8 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
```

Comment:
This is okay for the evaluation, but we need to use .NET 6 for the production articles.